<?xml version="1.0" ?>
<launch>

<!-- <include file="$(dirname)/franka_demo.launch" /> -->
<include file="$(dirname)/franka_control.launch" />
<include file="$(dirname)/camera_calibration.launch" />

<node name="save_pose_image" pkg="robot_control_VS"  type="franka_save_pose_jointangle_image"  output="screen"/>

</launch>